apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/GenuineCheckCancelModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Button, Flex, Popin, Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+
+export type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  onSkip: () => void;
+};
+
+const GenuineCheckCancelModal = ({ isOpen, onClose, onSkip }: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Popin position="relative" isOpen={isOpen}>
+      <Flex
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="space-between"
+        height="100%"
+        padding="40px"
+      >
+        <Text variant="h2" fontSize="20px" color="palette.warning.c80">
+          {t("syncOnboarding.manual.genuineCheckCancelModal.title")}
+        </Text>
+        <Text variant="paragraph" fontSize="25px" textAlign="center">
+          {t("syncOnboarding.manual.genuineCheckCancelModal.paragraph1")}
+        </Text>
+        <Text variant="paragraph" fontSize="18px" color="palette.neutral.c80" textAlign="center">
+          {t("syncOnboarding.manual.genuineCheckCancelModal.paragraph2")}
+        </Text>
+        <Flex flexDirection="column" width="100%">
+          <Button variant="main" width="100%" onClick={onClose} marginBottom="20px">
```

Comment:
Here you should use the spacing scale to set the margin bottom instead of hardcoding the pixel value: `marginBottom={7}`
https://react-ui-storybook.vercel.app/?path=/story/particles--spacing
cf. this conversation too https://ledger.slack.com/archives/C0278A6Q04D/p1660826691918959